### PR TITLE
opening-times (contact page) needs email contact form

### PR DESCRIPTION
**Pull request by malikbensalem**
Need to add contact form

### CODE REVIEW

File: app/controllers/contactController.scala [hunk 1]
```diff
@@ -0,0 +1,11 @@
+package controllers
+
+import javax.inject._
+import play.api.mvc._
+
+@Singleton
+class ContactController @Inject()(cc: ControllerComponents) extends AbstractController(cc) {
+  def contact = Action {
+    Ok(views.html.c("Your new application is ready."))
```
Comment (dannywaughman) [inline app/controllers/contactController.scala]: Ok(views.html.**c**("Your new application is ready."))
Fix typo ("c" -> "contact")

**Review by dannywaughman — CHANGES_REQUESTED** [review]
Fix the typo in comments, and rename contactController class to ContactController 
Add dependencies in build.sbt

**Review by RafSobolQAC — APPROVED** [review]
All OK.

**Review by dannywaughman — APPROVED** [review]
Looks good. Maybe change the name of contactController to ContactController for convention purposes

**Review by malikbensalem — COMMENTED** [review]
Merge fix

**Pull request merged**